analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/ProgramState.cs

```diff
@@ -135,7 +135,7 @@ public ProgramState ResetOperations()
             && other.PreservedSymbols.SetEquals(PreservedSymbols);
 
         public override string ToString() =>
-            Equals(Empty) ? "Empty" : SerializeException() + SerializeSymbols() + SerializeOperations() + SerializeCaptures();
+            Equals(Empty) ? "Empty" + Environment.NewLine : SerializeException() + SerializeSymbols() + SerializeOperations() + SerializeCaptures();
```

Comment:
In my oppinion, the consumer should decide if it needs a new line or not. Can't we do this in the caller?

Reply:
While I agree, I don't want to do it. It's a debug-only code. And we would have to trim the EOL in `Serialize` to add it back in the only usage it has.